### ISSUE TITLE
About BatchNorm and dropout

### ISSUE THREAD

**Issue by cham-3** [wy1iu/sphereface#28]
As I seen in your prototxt file, you didn't use neither batchnorm layer nor dropout. Why? 
Does such layers affect the network converging?
Thanks in advance.

**Comment by wy1iu** [comment]
BatchNorm actually helps convergence and you might need to use it when training a much deeper network (eg. ResNet-101) with A-Softmax. Dropout usually serves as regularization, but A-softmax also plays a similar role as regularization, which seems to have a little bit contradiction. Because we have not tried dropout, I can not comment too much about it. You are welcome to try more possibilities.

**Comment by Zhongdao** [comment]
I have tried dropout and found that it help avoid overfitting. With out dropout I will get overfit model sometimes.

**Comment by cham-3** [comment]
Thanks.
